pages/your_tasks/ethics.md

```diff
@@ -0,0 +1,116 @@
+    * How can harm to other beings and the environment be identified and mitigated in a timely manner?
+​
+### Solutions
+* Assess potential ethical implication through an ethics review
+    * Your local ethics committee can help you to review the ethical implications of the project or might guide you to more relevant bodies and resources
+* In order to address challenges when working with human data (see also the [RDMkit page on human data](human_data))
+    * Used standardised consent forms (see e.g. [GA4GH consent toolkit]({% raw %}https://www.ga4gh.org/resource-library/#{%22document%22:{%22category%22:%22Toolkit%20Component%22,%22related_products%22:%22Consent%20Toolkit%22}}{% endraw %}), [BBMRI](https://zenodo.org/record/3928446)) that can be represented in a machine actionable way ([DUO](https://www.ga4gh.org/product/data-use-ontology-duo/))
+    * Before you start collecting/processing data, be transparent about these in the consent form and also about the cases when withdrawal is no longer possible due to anonymization, aggregation, or other data processing.  Anticipate the possibility of consent/data withdrawal and implement administrative and technical processes.
+    * Data should be anonymized whenever possible (this is a non-reversible process), pseudonymisation (this is a reversible process) enhances data protection in cases where this is not possible (see also [Recommendations on shaping technology according to GDPR provisions](https://op.europa.eu/en/publication-detail/-/publication/a8e7a463-29c5-11e9-8d04-01aa75ed71a1/) and the [RDMkit data protection page](data_protection))
+    * Data analysis approaches that have potential to cause stigmatisation should be considered in advance and be discussed as part of an ethics review
+    * Create processes for incidental findings before you start collecting data, include the way the participant wants you to deal with it in the consent form
+* In order to manage the impact of data collection and sharing:
+    * The management of intellectual property rights connected with the data, if any, should be planned early on ([RDMkit data management plan page](dmp)), be part of collaboration/consortium agreements, and make use of [standard licensing terms](licensing)
+    * The Nagoya Protocol regulates access to genetic resources and conditions for transfer of genetic resources and traditional knowledge across counties. For implementations into national law please consult the [ABS Clearing House](https://absch.cbd.int/en/) (also see [RDMkit compliance monitoring & measurement](compliance))  
+    * Laws & Regulations concerning biosecurity, data export control and national interests might be linked from the [RDMkit national resources pages](national_resources)
+    * In order to ensure conformity with ethical research principles, the following should also be considered for data management:
+        * Follow general research ethics laws and guidelines (see below)
```

Comment:
Is it possible to link sub-section where it says 'see below'?

Reply:
```suggestion
        * Follow general research ethics laws and guidelines (see [below](ethics#how-can-i-identify-regulations-guidelines-and-laws-connected-to-ethics-in-my-research-context))
```

Reply:
@Nazeeefa added link